apps/ledger-live-desktop/src/renderer/modals/SignMessage/types.ts

```diff
@@ -0,0 +1,22 @@
+import { TFunction } from "react-i18next";
+import { Account } from "@ledgerhq/types-live";
+import { MessageData } from "@ledgerhq/live-common/hw/signMessage/types";
+import { TypedMessageData } from "@ledgerhq/live-common/families/ethereum/types";
+import { getNanoDisplayedInfosFor712 } from "@ledgerhq/live-common/families/ethereum/hw-signMessage";
+import { Step } from "~/renderer/components/Stepper";
+
+export type StepId = "summary" | "sign";
+
+export type StepProps = {
+  t: TFunction;
+  transitionTo: (str: string) => void;
+  account: Account;
+  error: Error;
+  message: MessageData | TypedMessageData;
+  onConfirmationHandler: (arg: any) => any;
```

Comment:
no way to type these ? Seems like it could be

```ts
onConfirmationHandler: (signature: string) => void;
```

Reply:
I could but I just changed that file to .ts to add the actual type that I'm sure of. 
The `onConfirmationHandler` is dependent on the `DeviceAction` typing which is (for now) still undefined as if you pass the `StepSign` into a `tsx` file, everything is `unknown`. 

I prefer not changing in this PR anything that is either 100% sure or at least related to this functionality.

<img width="504" alt="Screenshot 2022-11-04 at 16 50 51" src="https://user-images.githubusercontent.com/44363395/200019181-071ce6b0-40db-4349-90d5-aae7cb470b51.png">